gapis/atom/group.go

```diff
@@ -98,6 +119,12 @@ func (r Range) item(i uint64) GroupOrID     { return r.Start + ID(i) }
 func (r Range) itemIndex(i ID) uint64       { return uint64(i - r.Start) }
 func (r Range) split(i uint64) (Span, Span) { return r.Split(i) }
 
+func (c SubcommandRange) Bounds() Range               { return c.Range }
```

Comment:
So the interface documentation for `Bounds()` states this is an absolute range of command IDs. If I understand correctly, this is the range of the sub-commands. Please update the documentation.